Dashboard/models.py

```diff
@@ -205,6 +205,7 @@
     'pus': 'Pashto (پښتو)',
     'lzh': 'Classical Chinese (文言文)',
     'chv': 'Chuvash (Чӑвашла)',
+    'aeb': 'Tunisian Arabic',

```

Comment:
Should be listed as Arabic (Tunisian) for better ordering in lists. Also, would be good to add the name in Arabic script.

Non-blocking for now.

Reply:
Updated, thanks!